app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/CustomerSatisfactionPage.vue

```diff
@@ -105,6 +223,50 @@ const removeLabel = label => {
   }
 };
 
+// Check if template-related fields have changed
+const hasTemplateChanges = () => {
+  if (!isWhatsAppChannel.value) return false;
+
+  return (
+    originalTemplateValues.value.message !== state.message ||
+    originalTemplateValues.value.templateButtonText !==
+      state.templateButtonText ||
+    originalTemplateValues.value.templateLanguage !== state.templateLanguage
+  );
+};
```

Comment:
Let us extract local variable. 
```suggestion
const hasTemplateChanges = () => {
  if (!isWhatsAppChannel.value) return false;

  const original = originalTemplateValues.value;
  return (
    original.message !== state.message ||
    original.templateButtonText !== state.templateButtonText ||
    original.templateLanguage !== state.templateLanguage
  );
};
```